<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainView</class>
 <widget class="QMainWindow" name="MainView">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>569</width>
    <height>348</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>PyGUIBot</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <property name="acceptDrops">
    <bool>true</bool>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout">
    <property name="spacing">
     <number>3</number>
    </property>
    <property name="leftMargin">
     <number>6</number>
    </property>
    <property name="topMargin">
     <number>6</number>
    </property>
    <property name="rightMargin">
     <number>6</number>
    </property>
    <property name="bottomMargin">
     <number>6</number>
    </property>
    <item>
     <widget class="QLabel" name="status_label">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Expanding" vsizetype="Preferred">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QTreeWidget" name="commands_tree">
      <property name="acceptDrops">
       <bool>true</bool>
      </property>
      <property name="dragEnabled">
       <bool>true</bool>
      </property>
      <property name="dragDropMode">
       <enum>QAbstractItemView::InternalMove</enum>
      </property>
      <property name="selectionMode">
       <enum>QAbstractItemView::ContiguousSelection</enum>
      </property>
      <property name="selectionBehavior">
       <enum>QAbstractItemView::SelectItems</enum>
      </property>
      <property name="expandsOnDoubleClick">
       <bool>false</bool>
      </property>
      <column>
       <property name="text">
        <string>0</string>
       </property>
      </column>
      <column>
       <property name="text">
        <string>1</string>
       </property>
      </column>
      <column>
       <property name="text">
        <string>2</string>
       </property>
      </column>
      <column>
       <property name="text">
        <string>3</string>
       </property>
      </column>
      <column>
       <property name="text">
        <string>4</string>
       </property>
      </column>
      <item>
       <property name="text">
        <string>New Item</string>
       </property>
       <property name="text">
        <string>asdf</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>New Item</string>
       </property>
       <property name="text">
        <string>asdf</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>New Item</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>New Item</string>
       </property>
       <property name="text">
        <string>asgfsadf</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>New Item</string>
       </property>
       <item>
        <property name="text">
         <string>New Item</string>
        </property>
       </item>
      </item>
      <item>
       <property name="text">
        <string>New Item</string>
       </property>
       <item>
        <property name="text">
         <string>New Item</string>
        </property>
       </item>
      </item>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>569</width>
     <height>19</height>
    </rect>
   </property>
  </widget>
  <widget class="QToolBar" name="toolbar">
   <property name="windowTitle">
    <string>Abc</string>
   </property>
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
   <addaction name="open_button"/>
   <addaction name="separator"/>
   <addaction name="record_button"/>
   <addaction name="stop_button"/>
   <addaction name="run_button"/>
   <addaction name="separator"/>
   <addaction name="shift_left_button"/>
   <addaction name="shift_right_button"/>
   <addaction name="separator"/>
   <addaction name="comment_button"/>
   <addaction name="uncomment_button"/>
   <addaction name="separator"/>
   <addaction name="join_button"/>
   <addaction name="split_button"/>
  </widget>
  <action name="run_button">
   <property name="icon">
    <iconset>
     <normalon>../images/32/run.png</normalon>
    </iconset>
   </property>
   <property name="text">
    <string>Run</string>
   </property>
  </action>
  <action name="stop_button">
   <property name="icon">
    <iconset>
     <normalon>../images/32/stop.png</normalon>
    </iconset>
   </property>
   <property name="text">
    <string>Stop</string>
   </property>
  </action>
  <action name="record_button">
   <property name="icon">
    <iconset>
     <normalon>../images/32/record.png</normalon>
    </iconset>
   </property>
   <property name="text">
    <string>Record</string>
   </property>
  </action>
  <action name="comment_button">
   <property name="icon">
    <iconset>
     <normalon>../images/32/comment.png</normalon>
    </iconset>
   </property>
   <property name="text">
    <string>Comment</string>
   </property>
  </action>
  <action name="uncomment_button">
   <property name="icon">
    <iconset>
     <normalon>../images/32/uncomment.png</normalon>
    </iconset>
   </property>
   <property name="text">
    <string>Uncomment</string>
   </property>
  </action>
  <action name="shift_left_button">
   <property name="icon">
    <iconset>
     <normalon>../images/32/move-left.png</normalon>
    </iconset>
   </property>
   <property name="text">
    <string>Shift left</string>
   </property>
  </action>
  <action name="shift_right_button">
   <property name="icon">
    <iconset>
     <normalon>../images/32/move-right.png</normalon>
    </iconset>
   </property>
   <property name="text">
    <string>Shift right</string>
   </property>
  </action>
  <action name="open_button">
   <property name="icon">
    <iconset>
     <normalon>../images/32/open.png</normalon>
    </iconset>
   </property>
   <property name="text">
    <string>Open</string>
   </property>
  </action>
  <action name="join_button">
   <property name="icon">
    <iconset>
     <normalon>../images/32/join.png</normalon>
    </iconset>
   </property>
   <property name="text">
    <string>Join</string>
   </property>
  </action>
  <action name="split_button">
   <property name="icon">
    <iconset>
     <normalon>../images/32/split.png</normalon>
    </iconset>
   </property>
   <property name="text">
    <string>Split</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
